app/controllers/application_controller.rb

```diff
@@ -88,9 +88,27 @@ def skip_sentry_exception_types
   end
   # rubocop:enable Metrics/BlockLength
 
-  def set_uuid_tags
+  def set_tags_and_extra_context
     Thread.current['request_id'] = request.uuid
     Raven.extra_context(request_uuid: request.uuid)
+    Raven.user_context(user_context) if @current_user
+    Raven.tags_context(tags_context)
+  end
+
+  def user_context
+    {
+      uuid: @current_user&.uuid,
+      authn_context: @current_user&.authn_context,
```

Comment:
I can see an argument for this not being needed since it's collected in the tags context now

Reply:
I think only tags are going to aid in searchability / grouping.

I put the user context in there because we were adding that in as extra_context in a bunch of places. This way, when someone uses log_message_to_sentry they only add additional extra_context, and this base set of user specific ones will always exist. Kind of useful to have a base set of attributes for all sentry messages.

Reply:
oh, i see what you're saying. Yea... might not be needed. The tag one is slightly more useful in that it distinguishes signed-in user vs not signed-in user. Lets play with this though. I don't know how tags will appear in the message itself.

Reply:
Do you prefer I remove it?